spec/lib/sarif/bundle_audit_spec.rb

```diff
@@ -84,36 +85,44 @@
       it 'should return valid sarif report' do
         report = Salus::Report.new(project_name: "Neon Genesis")
         report.add_scan_report(scanner.report, required: false)
-        result = JSON.parse(report.to_sarif)["runs"][0]["results"][0]
-        rules = JSON.parse(report.to_sarif)["runs"][0]["tool"]["driver"]["rules"][0]
 
-        if rules['id'] == 'CVE-2021-22885'
+        results = JSON.parse(report.to_sarif)["runs"][0]["results"]
+        result = results.detect { |r| r["ruleId"] == 'CVE-2021-22885' }
```

Comment:
since you reference it twice might be nice to store in a var

Reply:
(CVE-2021-22885)

Reply:
Fixed, thanks!